### PR TITLE
fix: return responder for data

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

#### before

image

#### after

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

**Review by jungeun-cho — COMMENTED** [review]
scatter 시리즈에 테마로 hover에 `fillColor`배경색을 설정해줬는데 제대로 표시되지 않습니다.
확인부탁합니다.

```js
export const theme = () => {
  const { el } = createChart(genderHeightWeightData, {
    series: { selectable: true },
    theme: {
      series: {
        size: 8,
        borderWidth: 1,
        select: {
          size: 14,
          borderWidth: 4,
          borderColor: '#FFE662',
          fillColor: '#C23B22',
        },
        hover: {
          size: 14,
          borderWidth: 4,
          borderColor: '#FFE662',
          fillColor: '#ff6961',
        },
      },
    },
  });

  return el;
};
```

스크린샷 2021-01-04 17 09 05

**Comment by jung-han** [comment]
@jungeun-cho 수정된거 확인 부탁드립니다.

File: src/component/scatterSeries.ts [hunk 1]
```diff
@@ -158,8 +156,16 @@ export default class ScatterSeries extends Component {
...
+  private getResponderAppliedTheme(
+    closestModel: CircleResponderModel[],
+    type: RespondersThemeType
+  ) {
+    return closestModel.map((m) =>
+      deepMergedCopy(m, {
+        ...this.theme[type],
+        color: this.theme[type].fillColor,
+      })
+    );
```
Comment (jungeun-cho) [inline src/component/scatterSeries.ts:168]: theme에서 hover. select에 `size`도 입력하여 radius를 변경해줄 수 있어요.
`size`가 반영되고 있지 않아요.

스크린샷 2021-01-05 13 45 46

Comment (jung-han) [reply src/component/scatterSeries.ts]: 수정되었습니다.

Comment (jungeun-cho) [reply src/component/scatterSeries.ts]: `size / 2`만 확인 부탁드려요.

**Review by jungeun-cho — APPROVED** [review]
확인했습니다. 고생많았어요~

**Pull request merged**